src/features/YieldVaultDetails/components/yieldVaultDetails.tsx

```diff
@@ -92,6 +98,13 @@ export default function OpportunityDetails(): JSX.Element {
           />
         </div>
       </div>
+      <div className={styles.actions}>
+        <a href={addLiquidityUrl} target='_blank' className={styles.actions}>
+          <button className={styles.primaryButton} disabled={false}>
+            <div className={styles.buttonText2}>Get LP Tokens</div>
+          </button>
+        </a>
+      </div>
```

Comment:
UI folks take note here, I just threw this in. This is the button to link to the website to acquire LP tokens, I did not do anything for styling other than copying the "Deposit" button styling. If this belongs somewhere else on the UI or the styling should change, it should be part of some future work.